tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<commonviews:ToolPage
+    x:Class="DevHome.RepositoryManagement.Views.RepositoryManagementMainPageView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:commonCustomControls="using:DevHome.Common.Environments.CustomControls"
+    xmlns:commonviews="using:DevHome.Common.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:DevHome.RepositoryManagement.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:viewmodels="using:DevHome.RepositoryManagement.ViewModels"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
+    mc:Ignorable="d">
+    <commonviews:ToolPage.Resources>
+        <ResourceDictionary>
+            <Style x:Key="ListViewItemStretchStyle" TargetType="ListViewItem">
+                <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+                <Setter Property="Margin" Value="0" />
+                <Setter Property="Padding" Value="0" />
+            </Style>
+            <Style x:Key="HorizontalThreeDotsStyle" TargetType="Button">
+                <Setter Property="Background" Value="{ThemeResource ControlFillColorTransparent}" />
+                <Setter Property="Content" Value="&#xE712;" />
+                <Setter Property="FontFamily" Value="{StaticResource SymbolThemeFontFamily}" />
+                <Setter Property="HorizontalAlignment" Value="Right" />
+                <Setter Property="HorizontalAlignment" Value="Right" />
+                <Setter Property="BorderThickness" Value="0" />
+                <Setter Property="Width" Value="50" />
+                <Setter Property="Height" Value="36" />
+                <Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+            </Style>
+        </ResourceDictionary>
+    </commonviews:ToolPage.Resources>
+    <Grid Margin="10,0,20,0" RowSpacing="25">
+        <Grid.Resources>
+            <!--  Three Dot Button template  -->
+            <DataTemplate x:Key="ThreeDotsButton">
+                <Button x:Uid="MoreOptionsButton" Style="{StaticResource HorizontalThreeDotsStyle}">
+                    <Button.Flyout>
+                        <MenuFlyout>
+                            <MenuFlyoutItem Text="Open in File Explorer" />
+                            <MenuFlyoutItem Text="Open in Command Line" />
+                            <MenuFlyoutItem Text="Move Repository" />
+                            <MenuFlyoutItem Text="Add to Config file" />
+                            <MenuFlyoutItem Text="Run to config file" />
+                            <MenuFlyoutItem Text="Remove from this list" />
+                            <MenuFlyoutItem Text="Delete Repository" />
+                        </MenuFlyout>
+                    </Button.Flyout>
```

Comment:
Does this need to be repeated here and down below?

Reply:
It does not.  I believe this is a hold off from when I added it to the resources.  I removed the dupe from Grid.Resources.